client/ayon_core/hosts/maya/plugins/load/load_as_template.py

```diff
@@ -0,0 +1,39 @@
+from ayon_core.lib import (
+    BoolDef
+)
+from ayon_core.pipeline import (
+    load,
+    registered_host
+)
+from ayon_core.hosts.maya.api.workfile_template_builder import (
+    MayaTemplateBuilder
+)
+
+
+class LoadAsTemplate(load.LoaderPlugin):
+    """Load workfile as a template """
+
+    product_types = {"workfile"}
```

Comment:
```suggestion
    product_types = {"workfile", "mayaScene"}
```
Think we should extend this to maya scene publishes as well, so you can publish a selection of placeholders from a workfile.